internal/protogen/parser.go

```diff
@@ -28,12 +23,12 @@ const (
 	listVarSuffix = "_list" // list variable name suffix
 )
 
-type bookParser struct {
+type sheetParser struct {
```

Comment:
Should not change this name, as it is really a book level parser, not sheet level.